plover/formatting.py

```diff
@@ -513,6 +514,9 @@ def _atom_to_action_spaces_before(atom, last_action):
             else:
                 action.replace = last_action.text
                 action.text = _upper(last_action.text)
+        elif (meta.startswith(META_CARRY_CAPITALIZATION) or
+              meta.startswith(META_ATTACH_FLAG + META_CARRY_CAPITALIZATION)):
+            action = _apply_carry_capitalize(meta, last_action, spaces_after=False)
```

Comment:
This seems weird: `spaces_after` is not a keyword argument. I would convert it to a keyword argument (default=`False`), and omit it here, for consistency with the way it's handled elsewhere (e.g. `_apply_currency`).